### PR TITLE
update production block to use iframe on `blocks` repo

### PR DESCRIPTION
**Pull request by Wattenberger**

### CODE REVIEW

File: src/components/file-block.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ export function FileBlock(props: Omit<AppInnerProps, "onReset" | "blockType">) {
     console.log(path);
     const importType = path.endsWith(".css") ? "inline" : "raw";
     const contents = await import(
-      // /* @vite-ignore */ `../../..${path}?${importType}`
-      /* @vite-ignore */ `../dist/file-block.js?raw`
+      /* @vite-ignore */ `../../..${path}?${importType}`
+      // /* @vite-ignore */ `../dist/file-block.js?raw`
     );
     console.log(contents);
```
Comment (mattrothenberg) [inline src/components/file-block.tsx:46]: Let's nix this log statement

File: package.json [hunk 2]
```diff
@@ -29,10 +40,11 @@
     }
   ],
   "dependencies": {
-    "@githubnext/utils": "^0.10.0",
+    "@githubnext/utils": "^0.11.0",
     "@loadable/component": "^5.15.0",
     "@octokit/rest": "^18.12.0",
     "git-url-parse": "^11.6.0",
+    "npm-watch": "^0.11.0",
```
Comment (mattrothenberg) [inline package.json:47]: dev dependency?

File: package.json [hunk 3]
```diff
@@ -47,7 +59,9 @@
     "@types/react-dom": "^17.0.0",
     "@vitejs/plugin-react": "^1.0.0",
     "autoprefixer": "^10.4.0",
+    "concurrently": "^6.4.0",
     "esbuild": "^0.13.14",
+    "nodemon": "^2.0.15",
```
Comment (mattrothenberg) [inline package.json:64]: Is this being used?

Comment (Wattenberger) [reply package.json]: all good catches, thanks! will make sure to check before next PR 😶‍🌫️

**Review by mattrothenberg — APPROVED** [review]

**Pull request merged**